modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -971,6 +1068,8 @@ struct Infer: public cv::detail::KernelTag {
         GAPI_Assert(uu.params.input_names.size() == in_metas.size()
                     && "Known input layers count doesn't match input meta count");
 
+        const auto &input_name = uu.params.input_names.at(0);
```

Comment:
is it used anywhere?

Reply:
Removed, thanks! https://github.com/opencv/opencv/pull/24938/commits/2f249a8acab382376637a71aaba2940544eebd45